tutorials/03_thermal_camera.md

```diff
@@ -167,20 +200,29 @@ You should see something similar to this:
 @image html files/thermal_camera/thermal_camera_demo.png
 
 The window in the top-left corner shows the thermal camera's output.
-Taking a look at the [SDF file](https://github.com/ignitionrobotics/ign-gazebo/blob/c3391b1b664d1ec2b931d9a4ac757bde33b2a27b/examples/worlds/thermal_camera.sdf) for this example shows that the shapes were assigned the following temperatures:
+Taking a look at the [SDF file](https://github.com/ignitionrobotics/ign-gazebo/blob/46adfe54af6aa0828f9d38da9fd578da22f2a0eb/examples/worlds/thermal_camera.sdf) for this example shows that the shapes were assigned the following temperatures:
 * sphere: 600 Kelvin
 * box: 200 Kelvin
 * cylinder: 400 Kelvin
 
+If we take a look at the rescue randy and samsung J8 [fuel models](https://app.ignitionrobotics.org/dashboard), we see that they each have the following temperature range:
+* 0 Kelvin minimum
+* 655.35 Kelvin maximum
```

Comment:
I'm thinking we should provide more realistic values for the models on Fuel to set an example for others to follow, e.g. I imagine the rescue randy may have a temperature range from room temperature to human body temperature.

Reply:
Updated models with realistic temperatures are being used in https://github.com/ignitionrobotics/ign-sensors/pull/79/commits/2c53bffa31eb3014499439cbdae9d5902fbd37ab